modules/n1ql/pages/n1ql-language-reference/delete.adoc

```diff
@@ -249,4 +269,41 @@ include::example$dml/delete-filter.n1ql[]
 ----
 include::example$dml/delete-filter.jsonc[]
 ----
+====
+
+.Delete with LIMIT and OFFSET
+====
+This example deletes a subset of the airlines with a country of "France'.
+First, you query to get a list of the airlines in France.
+
+[source,sqlpp]
+----
+include::example$dml/delete-offset-before.n1ql[]
+----
+
+There are 21 documents in this collection with `country="France"`.
```

Comment:
@simon-dew I did not include the SELECT id FROM... query that I used here or below, is it ok to assume?